packages/sdk/src/custodianApi/qredo/QredoClient.ts

```diff
@@ -43,11 +44,8 @@ export class QredoClient extends EventEmitter {
         body: `grant_type=refresh_token&refresh_token=${this.refreshToken}`,
       });
 
-      if (!response.ok) {
-        throw new Error(`Error fetching the access token. Status: ${response.status} ${response.statusText}`);
-      }
-
-      const data = (await response.json()) as IQredoAccessTokenResponse;
+      const contextMessage = "Error fetching the access token";
```

Comment:
Since this new `contextMessage` always refers to an "error context/error info", imo it should state so in it's name, like instead `contextErrorMessage`. Or is there any case where it will not refer to a context about an error?